docs/providers/aws/guide/serverless.yml.md

```diff
@@ -63,6 +63,7 @@ provider:
       key1: value1
       key2: value2
   deploymentPrefix: serverless # The S3 prefix under which deployed artifacts should be stored. Default is serverless
+  disableDefaultOutputExportNames: true # optional, allows to disable default behavior of generating export names for CloudFormation outputs
```

Comment:
We should reflect default here, which while being `null`, is better to communicate with `false` (we do like that in other similar cases)

Reply:
good suggestion, thank you